static/locales/en.client.json

```diff
@@ -264,7 +264,8 @@
         "Reload": "Reload",
         "Sorry, access to this document has been denied. [{{error}}]": "Sorry, access to this document has been denied. [{{error}}]",
         "You can try reloading the document, or using recovery mode. Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. It also disables formulas. [{{error}}]": "You can try reloading the document, or using recovery mode. Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. It also disables formulas. [{{error}}]",
-        "You do not have edit access to this document": "You do not have edit access to this document"
+        "You do not have edit access to this document": "You do not have edit access to this document",
+        "You can try reloading the document, or using recovery mode.               Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. It also disables formulas. [{{error}}]": "You can try reloading the document, or using recovery mode.               Recovery mode opens the document to be fully accessible to owners, and inaccessible to others. It also disables formulas. [{{error}}]"
```

Comment:
This is strange. Seems to be a consequence of a pair of formatting changes in 61bd064f73464f762a0343b34780b2ba82b5f900 and d894b60fd4cd1dcdac82e11f592105ab05dfd368 - @CamilleLegeron @JakubSerafin would either of you mind if I reverted the recovery message in DocPageModel.ts back to the way it was prior to https://github.com/gristlabs/grist-core/commit/61bd064f73464f762a0343b34780b2ba82b5f900?

Reply:
it's fine for me

Reply:
I think the change to the "recovery" text [here](https://github.com/gristlabs/grist-core/commit/d894b60fd4cd1dcdac82e11f592105ab05dfd368#diff-52c94a2e46803d1d07ee58cf1c35f79eabed53066db80ac2a7cb8dfe64fbd996) should be reverted to omit the leading spaces. The change in https://github.com/gristlabs/grist-core/commit/61bd064f73464f762a0343b34780b2ba82b5f900, I believe, is important -- as I understand, the translation keys don't get extracted when a string is made up of multiple JS string literals.

Reply:
Ah, that makes sense @dsagal. @JakubSerafin can you revert your formatting change? Was there actually any necessary change in app/client/models/DocPageModel.ts? It is good in general to avoid committing whitespace changes of unrelated code.

Reply:
https://github.com/gristlabs/grist-core/pull/594